packages/venia-concept/src/components/ProductOptions/toolTip.js

```diff
@@ -62,6 +62,11 @@ class Tooltip extends Component {
         clearTimeout(this.timeoutId);
 
         this.setState({ isShowing: true });
+
+        // Creates a timer to hide the tooltip after 2.5s
+        this.timeoutId = setTimeout(() => {
+            this.setState({ isShowing: false });
+        }, 2500);
```

Comment:
I think we need to assign this `setTimeout` to its own class variable (in other words, don't reuse `timeoutId`) because 

* This could lead to the tooltip "blinking" while hovering
* This could lead to a situation where we don't cleanup properly

Both of which occur because this instance of `setTimeout` is not cleared (`window.clearTimeout`).


To reproduce:
1. The tooltip target receives focus. `this.timeoutId` is set to a timer going off in 2.5 seconds.
1. The target loses focus (`onBlur` triggered). `this.timeoutId` is re-assigned and on the next tick of the event loop, the tooltip is hidden.
1. The target gains focus again before the original 2.5 seconds have expired. When that timer expires, the tooltip will change its state to `isShowing: false`, even though we're actively hovering the target and it should be shown.